lib/phoenix_swagger/plug/swaggerui.ex

```diff
@@ -168,7 +168,8 @@ defmodule PhoenixSwagger.Plug.SwaggerUI do
   def init(opts) do
     app = Keyword.fetch!(opts, :otp_app)
     swagger_file = Keyword.fetch!(opts, :swagger_file)
-    body = EEx.eval_string(@template, spec_url: swagger_file)
+    template = Keyword.get(opts, :template, @template)
```

Comment:
Looks pretty good. Please update the `Options` section of the doc string to include `template`.